src/Symfony/Component/Security/Acl/Permission/BasicPermissionMap.php

```diff
@@ -105,4 +108,12 @@ public function contains($permission)
     {
         return isset($this->map[$permission]);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getMaskBuilder()
```

Comment:
The PermissionMap and its MaskBuilder are closely related, and it would be nice if we could retrieve the MaskBuilder we use in our application via the PermissionMap service.
Today, the only way to do it is : $maskBuilder = new MyBundle\MyNamespace\MyMaskBuilder();
So the getMaskBuilder() in the PermissionMap service can guarantee that the MaskBuilder you got is the good one.